lib/cmd/cmd_common.ml

```diff
@@ -3,6 +3,8 @@ open Core_extended.Std
 open Async.Std
 open Deferred.Monad_infix
 
+exception Execution_failed of Error.t
```

Comment:
Its still used pretty ubiquitously in our stuff. Until we go through and change it all, I will keep using it. I went through the exercise of trying to migrate to polymorphic variants again a month or so ago and ran into problems. I am really open to the idea of you going through and  converting everything  :) I have failed twice now.